client/src/js/app.js

```diff
@@ -287,11 +287,11 @@ function bhimaconfig($routeProvider) {
     templateUrl: 'partials/profit_center/profit_center.html'
   })
   .when('/profit_center/center/', {
-    controller: 'profitCenterAnalyse',
+    controller: 'AnalysisProfitCenterController as AnalysisProfitCenterCtrl',
```

Comment:
You'll save us a lot of typing (and bandwidth!) if you alias this a `CenterCtrl` or something shorter.  This isn't much shorter!